### DISCUSSION TITLE
How can I scale something like this?

### DISCUSSION THREAD

**Discussion by carmandale** [pmndrs/react-spring#1262] [Support]
I love this - https://codesandbox.io/embed/n9vo1my91p - but how could I scale the whole thing down, to fit on a page, and still maintain its functionality. I tried putting the Viewpager into another div and scaling that div, but it didn't seem to work. thanks for any help.

Dale

**Comment by manfromanotherland** [comment]
Have a look at the `style.css`, you can tweak the `width` and `height` of the `#root > div > div` selector to customise the size. I recommend adding a class in your implementation, as your selector will be likely different from the demo.

I modified the example to show you how, hope that helps → https://codesandbox.io/s/vigilant-christian-lqhom?file=/src/styles.css
